chore: align README for non-prefixed packages (#2994)
vaadin/web-components: packages/progress-bar/README.md

# @vaadin/progress-bar

> A web component used to show the completion status of a task or process.

Reviewer: ```suggestion
A web component for showing the completion status of a task or process.
```
